examples/src/main/java/com/mapbox/navigation/examples/core/FasterRouteActivity.kt

```diff
@@ -287,7 +289,9 @@ class FasterRouteActivity : AppCompatActivity(), OnMapReadyCallback {
 
         override fun onSuccess(result: LocationEngineResult?) {
             result?.locations?.firstOrNull()?.let {
-                activityRef.get()?.mapboxMap?.locationComponent?.forceLocationUpdate(it)
+                if (activityRef.get()?.mapboxMap?.locationComponent?.isLocationComponentActivated == true) {
+                    activityRef.get()?.mapboxMap?.locationComponent?.forceLocationUpdate(it)
+                }
```

Comment:
```suggestion
                if (activityRef.get()?.mapboxMap?.locationComponent?.isLocationComponentActivated!!) {
                    activityRef.get()?.mapboxMap?.locationComponent?.forceLocationUpdate(it)
                }
```

Reply:
this won't work because the mapboxMap isn't initialized at that moment. It will crash.